test/unit/alignment/pairwise/edit_distance_unbanded_test_template.hpp

```diff
@@ -85,7 +85,7 @@ template <template <bool, typename...> typename edit_traits_type,
           typename align_cfg_t>
 auto edit_distance(database_t && database, query_t && query, align_cfg_t && align_cfg)
 {
-    auto align_cfg_with_score_type = align_cfg | seqan3::align_cfg::result{seqan3::with_score}; // TODO remove once score type can be configured separately.
+    auto align_cfg_with_score_type = align_cfg;
```

Comment:
But then this is not necessary. It does not makes sense to just copy it here. So you can work with the default align_cfg. Please update the part below to use the original align_cfg if we know already that it has an output configured.